cmd/pool/order.go

```diff
@@ -210,6 +210,44 @@ func promptForConfirmation(msg string) bool {
 	}
 }
 
+// parseMinChanAmount parses and validates the value for the `min_chan_amt`
+// parameter.
+func parseMinChanAmount(ctx *cli.Context, amt uint64) (btcutil.Amount, error) {
+	minChanAmt := btcutil.Amount(ctx.Uint64("min_chan_amt"))
```

Comment:
I don't think this function does much parsing of `min_chan_amt`. It might be a good idea if this function was
```
validateMinChanAmount(minChanAmount btcutil.Amount, amt btcutil.Amount, auctionType) error
```

and move the `getMinChanAmount` functionality into a separate function. And then it would make more sense that a validator accepts `amt` as an argument. And testing might be simpler.